perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -295,7 +303,7 @@ sub body : Path('body') : Args(1) {
 
     $c->stash->{body_id} = $body_id;
 
-    $c->forward( 'check_for_super_user' );
+    $c->forward( 'check_for_super_user' ) if $c->cobrand->moniker eq 'zurich';
```

Comment:
I think this means (currently) any from_body user could edit any body by knowing the URL. If above changes are made, then I think this still needs changing to make sure the body being dealt with is the appropriate one.